source/StepBro.Core/Parser/StepBroListener.Expression.cs

```diff
@@ -363,7 +364,22 @@ public override void ExitExpBinary([NotNull] SBP.ExpBinaryContext context)
             {
                 var op = BinaryOperators.BinaryOperatorBase.GetOperator(context.op.Type);
                 // TODO: Check if operator is returned
-                var result = op.Resolve(this, first, last);
+                SBExpressionData result;
+                try
+                {
+                    result = op.Resolve(this, first, last);
+                }
+                catch (NotImplementedException e)
+                {
+                    StringBuilder exceptionMessage = new StringBuilder();
```

Comment:
As the TODO says in line 366, the op should be checked.
This could easily catch any kind of exception, and add the info to a parsing error instead.